### PR TITLE
Math-magicians: fetch quotes from API

### PR DESCRIPTION
**Pull request by Ofarouq310**
## Summary of branch implementations:-

> - [x] Use hooks & useEffect to fetch that data.
> - [x] Loading & error states are handled.
> - [x] Quotes are displayed beside the calculator and also displayed in a separate page.

### CODE REVIEW

File: src/components/Quote.js [hunk 1]
```diff
@@ -1,14 +1,69 @@
...
+  const displayQuote = quotes.map((quote) => (
+    <>
+      <p key={1}>
+        {quote.quote}
       </p>
-      <p>- William Paul Thurston</p>
-    </section>
+      <p key={2}>
+        -
+        {' '}
```
Comment (GedeonTS) [inline src/components/Quote.js:41]: - As per the project requirements [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_fetch_data_from_api.md#:~:text=Import%20the%20new,as%20the%20Calculator), kindly make sure the quote is on the same page with the calculator.

Your design:
Screenshot from 2023-03-24 12-14-12

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @Ofarouq310 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- Good job fetching the API :+1: 
- Good job handling errors :+1: 
- Good job using the isLoading state :+1: 
- No linter errors :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Quote.js (hunk 1 above)
Comment (Ofarouq310) [reply src/components/Quote.js:41]: image

@GedeonTS 
I appreciate your efforts, sir, but, there is indeed a place for the quotes as required beside the calculator.

Also, in addition, the requirements indicate that there must be also a separate page for the quote.
Please refer to the sneak peek of the project below. 
https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/sneak_peek.md

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Ofarouq310, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark:  Clean and Organized code
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**